common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1067,8 +1067,11 @@ def check_problem(self, data, override_time=False):
             self.set_last_submission_time()
 
         except (StudentInputError, ResponseError, LoncapaProblemError) as inst:
-            log.warning("StudentInputError in capa_module:problem_check",
-                        exc_info=True)
+            if self.runtime.DEBUG:
+                log.warning(
+                    "StudentInputError in capa_module:problem_check",
+                    exc_info=True
+                )
```

Comment:
What kinds of things trigger this? Just if they put invalid input into the field, like letters instead of numbers, or a formula that doesn't parse?